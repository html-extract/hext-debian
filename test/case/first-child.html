<div class="first-div">
  <ul>
    <li class="first-li"></li>
    <li class="nope"></li>
    <li class="nope"></li>
    <li class="nope"></li>
    <li class="nope"></li>
    <li class="nope"></li>
    <li class="nope"></li>
  </ul>
</div>
