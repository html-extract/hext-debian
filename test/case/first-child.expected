{"class":"first-div"}
{"class":"first-li"}
